model/registry/cloudfoundry.yaml

```diff
@@ -0,0 +1,158 @@
+          variable `CF_INSTANCE_INDEX`.
+        examples:
+          - '0'
+          - '1'
+      - id: space.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the CloudFoundry space the application is running in.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.space_name`. This is the same value as
+          reported by `cf spaces`.
+        examples: ['my-space-name']
+      - id: space.id
+        type: string
+        stability: experimental
+        brief: >
+          The guid of the CloudFoundry space the application is running in.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.space_id`. This is the same value as
+          reported by `cf space <space-name> --guid`.
+        examples: ['218fc5a9-a5f1-4b54-aa05-46717d0ab26d']
+      - id: org.name
+        type: string
+        stability: experimental
+        brief: >
+          The name of the CloudFoundry organization the app is running in.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.org_name`. This is the same value as
+          reported by `cf orgs`.
+        examples: ['my-org-name']
+      - id: org.id
+        type: string
+        stability: experimental
+        brief: >
+          The guid of the CloudFoundry org the application is running in.
+        note: |
+          Application instrumentation should use the value from environment
+          variable `VCAP_APPLICATION.org_id`. This is the same value as
+          reported by `cf org <org-name> --guid`.
+        examples: ['218fc5a9-a5f1-4b54-aa05-46717d0ab26d']
+      - id: source.type
+        type: string
+        stability: experimental
+        brief: >
+          The type of the event source.
+        note: |
+          CloudFoundry generates log messages with different source types for an
+          applications. This indicates, whether the log was created from the
+          actual application, a background task, the central load balancer, or
+          the CloudFoundry runtime, e.g. during a deployment.
+
+          The `source_type` is contained as a tag of the [Loggegator v2 envelope](https://github.com/cloudfoundry/loggregator-api#v2-envelope).
+          It is also available in the syslog drain structured data as documented
+          in the [CloudFoundry Dev Guide](https://docs.cloudfoundry.org/devguide/deploy-apps/streaming-logs.html).
+          Application developers can access the value of the `source_type` in
+          the log stream generated by `cf logs <app-name>`, usually in square
+          brackets.
+        examples:
+          - 'APP/PROC/WEB/0'
+          - 'APP/TASK/jobscheduler-task-70013817-ab92-4f75-82d4-de8ae4dd64b4'
+          - 'RTR'
+          - 'STG'
+      - id: process.id
```

Comment:
is it similar to the generic [`process.pid`](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/process.md) attribute?

Reply:
This is really not a pid. It is a guid given by the CloudFoundry runtime. A CloudFoundry application can consist of multiple [processes](https://v3-apidocs.cloudfoundry.org/version/release-candidate/#processes), similar to a K8s pod, that can contain multiple containers. This guid marks the concrete process emitting the signals. Earlier, I mentioned the trio org, space, app. this is sufficient, when there is only one process in the app. If there are more, the trio extends to a quartet: org -> space -> app -> process